.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Lint and Test
+run-name: ${{ github.actor }} is linting and testing 🚀
+
+on: [push]
+
+jobs:
+  Lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: 💅 Lint
+        run: |
+          make lint
+  Test:
+    runs-on: ubuntu-latest
```

Comment:
we could add `windows-latest` to the matrix if we want to widen the testing a bit

Reply:
I can add it and see if anything breaks out the box - though I'd question the sanity of anyone who wants to run this on a windows machine :fire:

Reply:
indeed, no harm in expanding the suite though.